app/main/views/g_cloud.py

```diff
@@ -543,6 +548,24 @@ def view_project(framework_family, project_id):
     )
 
 
+@direct_award_public.route("/<string:framework_family>/start-project", methods=("GET",))
```

Comment:
Obviously it doesn't matter but could this go further up the file, it'd be nice if they somehow got more into the order in which we expect people to see them, in terms of aiding understanding of the code...

Reply:
Yeah mentally flipped a coin on this one and in the end preferred breaking as little `git blame` information as possible.

Reply:
wait i'm thinking of a different view...